tests/test_host_manager.py

```diff
@@ -56,6 +56,7 @@ def test_slice(host_slice, num_hosts, hosts):
     ), f"{len(hosts[host_slice])} != {num_hosts} for {host_slice}"
 
 
+# pylint: disable=pointless-statement
```

Comment:
@ssbarnea here the linter was throwing a `Statement seems to have no effect (pointless-statement)` error and I tried to change the hosts[host_slice] line to result = hosts[host_slice] and then assert result is none but thought I should confirm this if need to change it like this or we can just add #pylint comment over there